cron/polygon/standalone-runner.js

```diff
@@ -0,0 +1,252 @@
+const allVaults = Object.keys(addresses);
+
+const disableCron = vaultAddress => Object.keys(addresses).find(
+  key => (addresses[key].NewVault && addresses[key].NewVault.toLowerCase() === vaultAddress.toLowerCase())
+  && (addresses[key].doHardwork === false)
+);
+
+const onlyProfit = vaultAddress => Object.keys(addresses).find(
+  key => (addresses[key].NewVault && addresses[key].NewVault.toLowerCase() === vaultAddress.toLowerCase())
+  && (addresses[key].doHardwork === "onlyProfit")
+);
+
+const vaultIds = allVaults
+  .filter(vaultId => addresses[vaultId].NewVault)
+  .filter(vault => !disableCron(addresses[vault].NewVault));
+
+const vaults = vaultIds.map(vaultId => addresses[vaultId].NewVault);
+
+
+// input vault key and output next vault key in the list
+function findNextVaultKey(curVault) {
+  let id = vaultIds.findIndex( (element => element == curVault) );
+  let nextId;
+
+  if(id == vaultIds.length-1){
+    nextId = 0;
+  } else {
+    nextId = id + 1;
+  }
+  return vaultIds[nextId];
+}
+
+async function roughQuoteXInMATIC(xAmount, xAddress, xMATICLPPair) {
+  console.log(xAmount);
+  console.log(xAddress);
+  console.log(xMATICLPPair);
+
+  var x = new web3.eth.Contract(IERC20Abi, xAddress);
+  var matic = new web3.eth.Contract(IERC20Abi, addresses.WMATIC);
+
+  let xInPair = await x.methods.balanceOf(xMATICLPPair).call();
+  let maticInPair = await matic.methods.balanceOf(xMATICLPPair).call();
+
+  // Use a very rough calculation. This would be erroneous when the farmAmount is large.
+  // However this should be fine for our purposes.
+  let maticOut = (maticInPair / xInPair) * xAmount;
+
+  return maticOut;
+}
+
+// determines the gas price by taking the minimum of "locally set max gas" and the gas price returned from api
+async function getGasPrice() {
+    return 65000000000; // 10 gwei in BSC
+}
+
+// properly setup the txSenderInfo for sending
+async function formulateTxSenderInfo(sender) {
+  let submitGasPrice = await getGasPrice();
+  let nonce = await web3.eth.getTransactionCount(sender);
+
+  console.log("gasPrice: ", submitGasPrice);
+
+  const txSenderInfo = {gasPrice: submitGasPrice, gas: settings.gasLimit, nonce, from: sender};
+  return txSenderInfo;
+}
+
+// helper functions
+// wait for ms
+function sleep(ms) {
+  return new Promise((resolve) => setTimeout(resolve, ms));
+}
+
+// return a random number between min and max
+function getRandomInt(min, max) {
+  min = Math.ceil(min);
+  max = Math.floor(max);
+  return Math.floor(Math.random() * (max - min) + min); //The maximum is exclusive and the minimum is inclusive
+}
+
+let curVaultKey = nextVault.next_vault_key;
+var controller = new web3.eth.Contract(IControllerV1Abi, addresses.Controller);
+var profitShareAddr = addresses.ProfitShareTarget;
+let vaultAddress = addresses[curVaultKey].NewVault;
+let rewardPoolAddress = addresses[curVaultKey].NewPool;
+let skip = addresses[curVaultKey].Skip;
```

Comment:
please use `disableCron` instead of Skip for consistency.

Reply:
`skip` wasn't actually working, removed and changed to `disableCron`